polaris.shopify.com/content/foundations/foundations/about-polaris/index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: About Polaris
+description: Learn what the Polaris design system is and how it's used by designers and developers to build world-class Shopify admin experiences.
+keywords:
+  - design system
+  - system
+  - getting started
+---
+
+Polaris is the design system for the Shopify admin. It's used both internally at Shopify and externally by third party developers and designers. From accessibility to performance to design expertise, Polaris gives you the tools to build a world-class admin experience.
+
+## Shopify admin
+
+Merchants set up their store, configure settings, and manage their business using the Shopify admin. It includes core aspects of the merchant's Shopify business, including orders, products, and customers.
+
+The Shopify admin is available across web and native platforms. The Polaris design system is available on every platform the Shopify admin is built for.
+
+### Web
+
+The web version of the Shopify admin is available for desktop and mobile web. Polaris web components and relevant packages are built with responsiveness in mind and can be used at any screen size.
+
+### Native
+
+The Shopify admin is also available as iOS and Android apps. Polaris components are available as native UI elements built with React Native. Because the iOS and Android apps also make use of embedded web views for parts of the admin, Polaris web components are used in these cases.
```

Comment:
```suggestion
The Shopify admin is also available as iOS and Android apps. Polaris components are available as native user interface (UI) elements built with React Native. Because the iOS and Android apps also make use of embedded web views for parts of the admin, Polaris web components are used in these cases.
```